lib/ansible/modules/network/nxos/nxos_ntp_options.py

```diff
@@ -77,144 +77,36 @@
 '''
 
 RETURN = '''
-proposed:
-    description: k/v pairs of parameters passed into module
-    returned: always
-    type: dict
-    sample: {"logging": false, "master": true, "stratum": "11"}
-existing:
-    description:
-        - k/v pairs of existing ntp options
-    returned: always
-    type: dict
-    sample: {"logging": true, "master": true, "stratum": "8"}
-end_state:
-    description: k/v pairs of ntp options after module execution
-    returned: always
-    type: dict
-    sample: {"logging": false, "master": true, "stratum": "11"}
 updates:
```

Comment:
Might as well change this to `commands` so the documentation is consistent across modules